DI-435/find_vcfs_to_merge.py

```
@@ -48,6 +48,18 @@ def parse_args() -> argparse.Namespace:
     )
 
+    parser.add_argument(
+        "-r",
+        "--run_mode",
+        choices=['find_qc', 'no_qc'],
+        required=True,
+        help=(
+            "Runmode - either 'find_qc' where QC status files are "
+            "searched for and used to find failed samples, or 'no_qc'"
+            "where no QC status files are searched for"
+        )
```

Ruff (0.8.0)

Trailing comma missing

Add trailing comma

(COM812)